jwst/datamodels/schemas/combinedspec.schema.yaml

```diff
@@ -15,3 +15,9 @@ allOf:
       default: 1
       fits_keyword: SPORDER
       fits_hdu: COMBINE1D
+    source_id:
+      title: Source ID
+      type: integer
+      default: 0
```

Comment:
I notice this entry has a default value, while the one in multicombinedspec does not. I think we could leave out the default value.

Reply:
Thought I had removed those!